File: JS Essentials/6.0 Functions/04. Orders.js
function orders(item, price) {
    const itemsObj = {'coffee' : 1.50, 'water' : 1.00, 'coke':1.40, 'snacks':2.00}
    return (itemsObj[item] * price).toFixed(2)
}

// console.log(orders('water', 5))

File: Python Essentials/6.0 Functions/04. Orders.py
def shop(item_kind, count ):
    items_dict = {'coffee': 1.50, 'water': 1.00, 'coke': 1.40, 'snacks': 2.00}
    return f'{(items_dict[item_kind] * count):.2f}'


print(shop(input(), int(input())))



'''
water
5
#5.00
'''